c2corg_api/search/sync.py

```diff
@@ -62,13 +64,27 @@ def remove_doc():
             # https://github.com/c2corg/v6_api/issues/89
             DBSession.refresh(document)
 
+        is_user_profile = isinstance(document, UserProfile)
+        if is_user_profile:
+            # set user login + full-name as document title so that it can
+            # be searched
+            user_login, user_name = DBSession. \
+                query(User.name, User.username). \
+                filter(User.id == document.document_id). \
+                first()
+            user_title = '{0} {1}'.format(user_name or '', user_login or '')
```

Comment:
In the api the [user](https://github.com/c2corg/v6_api/blob/master/c2corg_api/models/user.py#L59-L60) has a field `name` and `username`. `name` is filled with the old `topo_name` and `username` with `login_name` (see https://github.com/c2corg/v6_api/blob/master/c2corg_api/scripts/migration/users.py#L58-L66).

> By the way, could we use ES indexing of the users to associated users to an outing?

We would have to store the usernames on the outing in ES.